#pwm1_vf_string: "[[15200.0, 0], [15230.0, 1.85], [15240.0, 2.0], [15250.0, 2.3], [15260.0, 2.6], [15280.0, 3.0]]"
pwm1_vf_string: "[[15200.0, 0], [15280.0, 1.00], [15320.0, 1.50], [15350.0, 2.3], [15380.0, 2.6], [15400.0, 3.0]]"
